### ISSUE TITLE
VPS stops working when DH Shadowmap is Enabled

### ISSUE THREAD

**Issue by joshua-igor** [X0nk/Bliss-Shader#252]
### Describe the bug
When you enable DH Shadowmap on the "Settings for supported mods" section, Variable Penumbra Shadow stops working completely, as if it was disabled.
With DH Shadowmap Disabled you can properly see softnened shadows on distant objests.
![Screenshot 2024-07-20 161152](https://github.com/user-attachments/assets/53f03b2c-6f03-45d1-ade4-66704b1d521f)
When you Enable DH Shadowmap without changing anything else, you can see that all shadows have now the same amount of penumbra.
![Screenshot 2024-07-20 161240](https://github.com/user-attachments/assets/432fd1ab-c7a2-4450-9835-bbf86c7f8f9f)

### Describe how your minecraft is set up
1. Include a screenshot of the in-game f3 debug overlay. Please make sure it is fully visible.
![image](https://github.com/user-attachments/assets/a7d27f2a-f602-414b-a4e2-584f59ed988f)

3. If you cannot do option 1. Please list the following information instead:
- What GPU vendor you are using (e.g Apple/AMD/Nvidia/Intel): NVIDIA RTX 3070
- What operating system you are using (e.g Windows/Linux/macOS): Windows 11
- What minecraft version you are playing (e.g 1.20.4, 1.16.5, 1.12): 1.21
- If you are using a special client for minecraft (e.g lunar, badlion, feather)
- What shader loading mod you are using (e.g Iris/oculus/optifine): Iris 1.7.3
- What version of the shader you are using (e.g Bliss v2.0.4, Bliss Shader Main): Bliss Commit #482
- If you are using any, what additional mods you are using besides the ones required to use the shader (e.g distant horizons, physics mod): Distant Horizons 2.1.2

**Comment by X0nk** [comment]
yeah just stop using that feature, it is currently broken on iris/distant horizons end for a few months and indefinitely
